src/content/docs/en/guides/migrate-to-astro/from-gitbook.mdx

```diff
@@ -0,0 +1,72 @@
+
+- GitBook stores your content in a database. In Astro, you will have individual files (typically Markdown or MDX) in your [project directory](/en/core-concepts/project-structure/) for each page's content. Or, you can choose to use a [CMS for your content](/en/guides/cms/), even your existing GitBook instance, and use Astro to fetch and present the data.
+
+- GitBook sites use a specialised version of Markdoc (a superset of Markdown) for content. Astro also supports Markdoc via the optional [Markdoc integration](/en/guides/integrations-guide/markdoc/) to use `.mdx` files in addition to standard Markdown files.
+
+
+## Switch from GitBook to Astro
+
+To convert a GitBook documentation site to Astro, start with our official docs theme starter template, or explore more community docs themes in our [theme showcase](https://astro.build/themes?search=&categories%5B%5D=docs).
+
+You can pass a `--template` argument to the `create astro` command to start a new Astro project with one of our official starters. Or, you can [start a new project from any existing Astro repository on GitHub](/en/install/auto/#starter-templates).
+
+  <PackageManagerTabs>
+    <Fragment slot="npm">
+    ```shell
+    npm create astro@latest -- --template docs
+    ```
+    </Fragment>
+    <Fragment slot="pnpm">
+    ```shell
+    pnpm create astro@latest --template docs
+    ```
+    </Fragment>
+    <Fragment slot="yarn">
+    ```shell
+    yarn create astro --template docs
+    ```
+    </Fragment>
+  </PackageManagerTabs>
+
+Once you have a new Astro project, you can sync your existing GitBook content to your new Astro project. GitBook has a [gitsync feature](https://docs.gitbook.com/product-tour/git-sync) that will automatically sync your GitBook content to a GitHub/GitLab repository. To sync directly to the docs template's content collection, specify `src/content/docs/en` or `src/content/docs` as the project directory.
+
+:::caution
+When enabling git sync be sure to specify "**GitBook to GitHub**" as the priority. This will ensure that your GitBook content is synced to your GitHub repository. Otherwise, you will overwrite your existing GitBook content.
+![Choose Priority](/guides/gitbook-gitsync.png)
+:::
+
+After syncing the content to your repository and deleting the sync, you'll have a copy of your GitBook content in your repository. There are a few things to take into account before you can use this content in your Astro project:
+
+- Gitbook uses a special syntax that differs from Markdoc where the `/` prefix denoting a closing tag is replaced with `end` for Gitbook files. In addition, some features of Gitbook rely on custom Markdoc components.
+- Astro's [Markdoc integration](/en/guides/integrations-guide/markdoc/) requires that the file extension be `.mdoc`. This is to avoid conflicts with other Markdown extensions like `.mdx` and `.md`.
```

Comment:
```suggestion
Once you have a new Astro project, you can sync your existing GitBook content to your new Astro project. GitBook has a [gitsync feature](https://docs.gitbook.com/product-tour/git-sync) that will automatically sync your GitBook content to a GitHub/GitLab repository. 

To sync directly to the docs template's content collection, specify `src/content/docs/en` or `src/content/docs` as the project directory.

:::caution
When enabling git sync be sure to specify "**GitBook to GitHub**" as the priority. This will ensure that your GitBook content is synced to your GitHub repository. Otherwise, you will overwrite your existing GitBook content.
:::

After syncing the content, you will now have a copy of your GitBook content in your Astro repository. Disable git sync to prevent future syncing with GitBook.

Note that although you now have your content migrated to your Astro project, it will not be immediately usable. To use this content in your Astro site, you will need to spend some time manually changing Gitbook's Markdoc into a format compatible with Astro. In particular:

- Astro's [Markdoc integration](/en/guides/integrations-guide/markdoc/) requires that the file extension be `.mdoc`. This is to avoid conflicts with other Markdown extensions like `.mdx` and `.md`.
- Gitbook uses a special syntax that differs from Markdoc where the `/` prefix denoting a closing tag is replaced with `end` for Gitbook files. You will need to update this notation throughout your files.
- Some features of Gitbook rely on custom Markdoc components. These components will not exist in Astro and must be deleted from your Markdoc code.
- The git sync only syncs your content, not any custom components you may have created. You will need to rebuild any components as Astro components, or UI framework components.
```
FAIR WARNING: The above is "Sarah AI" - may not actually be correct, but sounds good!!  :sweat_smile:

 "deleted the sync" sounds a little odd to me, so I just wrote placeholder content above with what I think is something like the correct wording.

Also, I wanted to emphasis "you're gonna have to do stuff!! This will not just work!!" so I added more to the intro to the points to consider. And, I changed the sense of "things to be aware of" to try to make them "things to do." In the process, I might have totally made up the last (new) point about components? :sweat_smile:  (I'm not sure whether people DO create their own components in GitBook??) but if so, that's the sort of thing we should be saying!  :smile:

Reply:
Sounds good to me! Custom components I don't believe are possible in GitBook at the moment, so I've left that final bullet point out, but this reads much better, thanks!